cypress/support/pages/Cart.pageObject.js

```diff
@@ -0,0 +1,55 @@
+import PageObject from '../PageObject';
+
+class CartPageObject extends PageObject {
+  CartProducts(productName) {
+    cy.get('.table')
+      .should('contain', productName);
+  }
+
+  PlaceOrderButton() {
```

Comment:
Method names must contain action or assert in the name.
PlaceOrderButton => clickOnOrderBtn
TypeName => fillNameField
OkButton => clickOnOKBtn